src/main.cpp

```diff
@@ -38,12 +38,17 @@ char power_path[60] = "";
 char power_l1_path[60] = "";
 char power_l2_path[60] = "";
 char power_l3_path[60] = "";
-char energy_in_path[60] ="";
-char energy_out_path[60] ="";
+char energy_in_path[60] = "";
+char energy_out_path[60] = "";
 char shelly_mac[13];
 char shelly_name[26] = "shellypro3em-";
 char query_period[10] = "1000";
 
+unsigned long ledOffTime = 0;
+int led = 0;
+const uint8_t ledblinkduration = 50;
+char led_gpio[2] = "";

```

Comment:
if GPIO > 9 this would fail, as we need terminating character in this string as as well, so change to 3